### ISSUE TITLE
Help with terms

### ISSUE THREAD

**Issue by linkedList123** [DeepLabCut/DeepLabCut#1610]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

While working with DLC, I was just wondering what the following losses in the learning stat csvs measure: Loss, locref loss, scmap loss, limb loss, and lr. 

Also, what distinguishes resnet from efficietnet and mobilenet?

Lastly, what exactly does the scmap/locref plots when evaluating network? What are PAFs and what do they measure?

Thanks!

### Operating System

operating system

### DeepLabCut version

dlc version

### DeepLabCut mode

single animal

### Device type

gpu

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by KonradDanielewski** [comment]
I will try to give a not too technical explanation which might help to get a feel of what all those things are- if you feel that's lacking or is imprecise please correct me. 

You can look at loss in terms of how far away from optimal solution to the problem the model you're training is - for instance your loss function can be mean squared error which would calculate average squared difference between estimated value and the actual value (how close model placed the label in relation to where you placed the label). Locref stands for location refinement layer, scmap - score map, PAF is a part affinity field (which I think in terms of training stats is named limb as it encodes pairwise relationships between bodyparts) . That being said, location refinement tries to find position of bodyparts based on their relation to other bodyparts, score maps look for parts of image that have a wanted feature (it helps to understand when you plot them at the evaluation step), PAF - explained in the brackets before. 

LR stands for learning rate which is a parameter controlling how strongly the weights should shift in pursue of the most optimal solution - hence why at first lr is high (the model gets close to finding where the bodyparts are faster) but then gets smaller to slowly adjust the weights so the model can get as precise as possible (think of it as change in weights shifting the placement of your label, first by 10s of pixels but then on a pixel level to find just the right spot).  

When it comes to architectures themselves it's something that is very complicated but generally can be summed up to different solutions to achieve as best result as possible under certain circumstances. 
ResNets are Residual Neural Networks and they utilize this cool feature of having skip connections which help to solve the problem of vanishing gradient - when changes made to weights become too small to actually change anything. Very precise, not super computationally expensive, generally a good choice for most of the users. 
MobilNets have been created for the purpose of running low latency inference (analysis) on - you guessed it - mobile devices, which are constrained by their low computational power and aim at power efficiency. They are great for online (live) inference and can easily be used even on low end hardware to do just that - for instance if your experiment requires you to track the paw of your rat in lever pressing experiment and you want to start optogenetic excitation when the animal touches the lever.
EfficientNets I understand the least of all those but from what I understand they work with compund scaling in mind, meaning instead of trying to make the architecture deeper or wider or using higher resolution images we can find a solution that's a sweet spot for all of those allowing us to get the best performance with least computational power - though from what I understand the problem here is the way they work is actually not optimal for how GPUs process data (albeit this is actually true for image processing generally as we try to move huge amounts of data with not that much computation done on this data).

Hope this helps to clear things up a little and I didn't make any horrible mistakes here.

**Comment by MMathisLab** [comment]
@deplabcut also note all terms are explained in the code docstring and in the papers; if you look at our work here, see the methods: https://www.biorxiv.org/content/10.1101/2021.04.30.442096v1
